crates/core_arch/src/x86/sse.rs

```diff
@@ -3658,6 +3672,13 @@ mod tests {
         assert_eq_m128(r, e);
     }
 
+    #[simd_test(enable = "sse2")]
+    unsafe fn test_mm_loadu_si64() {
+        let a = _mm_set_epi64x(5, 0);
```

Comment:
In memory this is represented as `[0, 0, 0, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0]`, not `[5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]` as you expected.

Reply:
Oh, now it still failed with

```rust
        let a = _mm_set_epi8(0, 0, 0, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0);
        let r = _mm_loadu_si64(&a as *const _ as *const _);
        assert_eq_m128i(a, r);
```

Or did I declare `[0, 0, 0, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0]` wrongly, or is it endianness issue?

```
thread 'core_arch::x86::sse::tests::test_mm_loadu_si64' panicked at 'assertion failed: `(left == right)`
  left: `[360287970189639680, 0]`,
 right: `[0, 360287970189639680]`', crates/core_arch/src/x86/test.rs:20:5
```

Reply:
I would recommend testing the equivalent C code.